cartographer_ros/cartographer_ros/node.cc

```diff
@@ -459,7 +459,10 @@ void Node::FinishAllTrajectories() {
 
 void Node::FinishTrajectory(const int trajectory_id) {
   carto::common::MutexLocker lock(&mutex_);
+  CHECK(is_active_trajectory_.count(trajectory_id) > 0 &&
```

Comment:
And if you want to better document that we expect the value to already exist you could write it like this:
```
CHECK(is_active_trajectory_.at(trajectory_id));
```